README.md

```diff
@@ -83,6 +93,14 @@ To do this follow the instructions in the
 * **Sparse**, may be installed using command `pip install 'qiskit-machine-learning[sparse]'` to install the
   package. Sparse being installed will enable the usage of sparse arrays/tensors.
 
+## Migration to Qiskit 1.x
+> [!NOTE]
```

Comment:
You might like to add some text like perhaps to explain a little

>Qiskit Machine Learning learning depends on Qiskit which will be automatically installed as a dependency when you install Qiskit Machine Learning. If in your environment you have a pre-1.0 version of Qiskit installed, however that was installed, and want to upgrade to 1.0 you should take note of the following:

Reply:
Thanks! This is a useful addition to the guide